website/_dogfooding/testSwizzleThemeClassic.mjs

```diff
@@ -50,7 +53,12 @@ console.log('\n');
 
 await fs.remove(toPath);
 
-let componentNames = await readComponentNames(themePath);
+async function getAllComponentNames() {
+  const names = await readComponentNames(themePath);
+  return names.concat(await getMissingIntermediateComponentFolderNames(names));
+}
+
+let componentNames = await getAllComponentNames();
```

Comment:
Super nit but since we are using `let` anyway I don't think we need this function